.github/workflows/voyager-debezium-workflow.yml

```diff
@@ -65,19 +65,21 @@ jobs:
       - name: Build Debezium Server
         run: |
           sh buildDebeziumServer.sh
+          cd yb-client-wrapper/
+          mvn clean package
       - name: Package Debezium Server
         run: |
+          # package yb-client-wrapper.jar into debezium-server
```

Comment:
I think we should name the package(and dir) yb-client-cdc-stream-wrapper because this is very specific to cdc streams not a generic client wrapper :)